pyvista/plotting/plotter.py

```diff
@@ -5433,35 +5552,35 @@ def remove_scalar_bar(self, *args, **kwargs) -> None:  # numpydoc ignore=PR01,RT
 
     def add_point_labels(
         self,
-        points,
-        labels,
-        italic=False,
-        bold=True,
-        font_size=None,
-        text_color=None,
-        font_family=None,
-        font_file=None,
-        shadow=False,
-        show_points=True,
-        point_color=None,
-        point_size=None,
-        name=None,
-        shape_color='grey',
-        shape='rounded_rect',
-        fill_shape=True,
-        margin=3,
-        shape_opacity=1.0,
-        pickable=False,
-        render_points_as_spheres=False,
-        tolerance=0.001,
-        reset_camera=None,
-        always_visible=False,
-        render=True,
-        justification_horizontal=None,
-        justification_vertical=None,
-        background_color=None,
-        background_opacity=None,
-    ):
+        points: Sequence[float] | NumpyArray[float] | pyvista.DataSet | _vtk.vtkAlgorithm,
+        labels: list[str | int] | str,
+        italic: bool = False,
+        bold: bool = True,
+        font_size: float | None = None,
+        text_color: ColorLike | None = None,
+        font_family: Literal['courier', 'times', 'arial'] | None = None,
```

Comment:
This `Literal` is repeated for `add_text`, can we use an alias instead so both use the same type hint, e.g. `FontFamilyOptions  = Literal['courier', 'times', 'arial']`
```suggestion
        font_family: FontFamilyOptions | None = None,
```